bot/extensions/vote_cog.py

```diff
@@ -0,0 +1,315 @@
+	async def vote(self, ctx: Context, *, title: str, options_count: int = 2, poll_time: int = 120) -> None:
+		if options_count < 2:
+			await ctx.interaction.response.send_message('Only 2 or more options is allowed.', ephemeral=True)
+			return
+		cancel_keywords = ['!abort']
+
+		# if not title:
+		# 	# Create title embed
+		# 	title_embed = Embed(
+		# 		color=self.bot.default_color,
+		# 		title='POLL CREATION INFO',
+		# 		description='**Input poll title**'
+		# 	)
+		# 	# Send a DM to the command executor
+		# 	await ctx.author.send(embed=title_embed)
+
+		# 	# TODO: Think if input title can be invalid
+		# 	title = await self.bot.wait_for('message', check=self.message_check(ctx.author.dm_channel))
+
+		# 	# Check if user cancelled the poll
+		# 	if title.content.lower().strip() in cancel_keywords:
+		# 		await ctx.interaction.response.send_message('Aborted', ephemeral=True)
+		# 		return
+
+		options_embed = Embed(
+			color=self.bot.default_color,
+			title='POLL CREATION INFO',
+			description=f'**Input {options_count} poll options.**\n To abort the poll type: **!abort**'
+		)
+
+		await ctx.author.send(embed=options_embed)
+		options = []
+
+		# Read the input from dm and save it in options list
+		for index in range(options_count):
+			option_embed = Embed(
+				color=self.bot.default_color,
+				title='POLL CREATION INFO',
+				description=f'**Input option {index + 1}.**'
+			)
+			await ctx.author.send(embed=option_embed)
+			# Wait for input
+			option = await self.bot.wait_for('message', check=self.message_check(ctx.author.dm_channel))
+			# Check if user cancelled the poll
+			if option.content in cancel_keywords:
+				await ctx.author.send('**Successfully aborted!**')
+				return
+			options.append(option.content)
+
+		# Send a success message
+		await ctx.author.send(f'Great! The poll: {title} with {len(options)} options created! On channel: {ctx.channel.name}')
+		if options_count == 2:
+			emojis = [
+				'👍', '👎'
+			]
+		else:
+			emojis = [
+				'🟥', '🟧', '🟨', '🟩', '🟦', '🟪', '⬛', '⬜', '🟫', 
+				'🔴', '🟠', '🟡', '🟢', '🔵', '🟣', '⚫', '⚪', '🟤',
+			]
+		# Votes counter according to vote emoji chosen
+		counter = {}
+		# allowed size of counter array, needed if there is more than 18 options.
+		allowed_emojis_size = len(options) if len(options) < len(emojis) else len(emojis)
+		for emoji_index in range(allowed_emojis_size):
+			counter[emojis[emoji_index]] = 0
+
+		# Create poll embed
+		poll_embed = PollEmbed(
+			options=options, 
+			emojis=emojis, 
+			counter=counter, 
+			title=title
+		)
+		# Create poll view
+		view = PollView(emojis, allowed_emojis_size, poll_embed)
+		# build/initialize embed
+		poll_embed.build()
+
+		# Send a poll message
+		poll = await ctx.channel.send(embed=poll_embed, view=view)
+		# Create timer
+		timer = Timer(poll, poll_time, poll_embed, view)
+		await timer.start()
+		
+		# Output the winner option
+		await poll_embed.get_and_print_winner(ctx)
+
+
+
+
```

Comment:
4 new lines instead of 2.

Reply:
> I might have missed some other thing, but I will let @PenguinBoi12 double-check. If you could fix the couple of comments I made, it will already be better. Thank you for your work.

Thank you for your remarks! Will make everything as you said.